tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/addressing/AddressingTest.java

```diff
@@ -268,6 +268,62 @@ public void testPurgeOnNoConsumersFalse() throws Exception {
       Wait.assertEquals(1, server.locateQueue(queueName)::getMessageCount);
    }
 
+
+   @Test
+   public void testQueueEnabledDisabled() throws Exception {
```

Comment:
@clebertsuconic see here this is the actual functionality test, where enabling / disabling the queue.

(other tests added are just around checking config works and updates etc should reload etc occur)